src/components/Header/index.tsx

```diff
@@ -0,0 +1,94 @@
+import Logo from "./logo"; 
+import DesktopNav from "./Devices/desktop";
+import MobileNav from "./Devices/mobile";
+import { HamburgerIcon, CloseIcon } from "@chakra-ui/icons";
+import { Box, Collapse, Flex, IconButton, useDisclosure } from "@chakra-ui/react";
+
+/* 1. Header Props */
+
+export interface HeaderProps {
+  name: string;
+  subLabel?: string;
+  children?: Array<HeaderProps>;
+  href?: string;
+}
+
+/* 2. Object with Header options */
+
+export const Links: Array<HeaderProps> = [
+
+  {
+    name: "Create",
+    href: "/create"
+  },
+
+  {
+    name: "Explore",
+    href: "/explore"
+  },
+
+  {
+    name: "Connect Wallet",
```

Comment:
@kevinjung13 good idea but connect wallet we will replace with https://www.rainbowkit.com/
do you feel capable to integrate it?